lib/tasks/case_contact_types_reminder.rb

```diff
@@ -0,0 +1,83 @@
+class CaseContactTypesReminder
+  FIRST_MESSAGE = "It's been 60 days or more since you've reached out to these members of your youth's network:"
+  THIRD_MESSAGE = "If you have made contact with them in the past 60 days, remember to log it: [link to create new case contact for assigned case]"
+
+  def send!
+    responses = []
+    eligible_volunteers = User.where(type: :Volunteer, receive_sms_notifications: true).where.not(phone_number: nil)
+    eligible_volunteers.each do |volunteer|
+      next if last_reminder_within_quarter(volunteer)
```

Comment:
is "next if" similar to continue statements? So if this function "last_remainder ... " returns true, the loop skips to the next volunteer?

Reply:
Yes that is correct